src/TinyCiv.Client/Code/units/Cavalry.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Media;
+using TinyCiv.Shared.Game;
+
+namespace TinyCiv.Client.Code.units
```

Comment:
```suggestion
namespace TinyCiv.Client.Code.Units
```
C# naming convention